### PR TITLE
Do not do negative IGV coords.

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Negative IGV coordinates causes view to hang. See PR #2356 for discussion.

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
@dnil what interval should  search for instance?

**Comment by dnil** [comment]
Try eg that variant from the other PR thread; https://scout-stage.scilifelab.se/cust000/643594-300M/sv/variants/1b8aa31ab526f4ef5d1181f523373414
stalls on master but should be ok on branch. I can test as well, but later..

**Comment by dnil** [comment]
Ok, looks good:
Screenshot 2021-02-15 at 09 38 54
Let me just check one more thing..

File: scout/server/blueprints/variant/templates/variant/utils.html [hunk 1]
```diff
@@ -272,17 +272,27 @@
...
-            <input type="hidden" name="stop" value="{{variant.end + 500}}">
+            {% set initial_view.start = variant.end - 500 %}
+          {% endif %}
+
+          {% if initial_view.start<1 %}
+            {% set initial_view.start = 1 %}
           {% endif %}
+
+          {% if initial_view.end<1 %}
+            {% set initial_view.end = 1 %}
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/utils.html:291]: 👍

**Review by northwestwitch — APPROVED** [review]
This code is much better!

File: scout/server/blueprints/variant/templates/variant/utils.html (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/utils.html]: 😊

Comment (dnil) [reply scout/server/blueprints/variant/templates/variant/utils.html]: Better, just off by one - though it is a lot easier to see and correct now!

Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/utils.html]: It is!

**Pull request merged**